### DISCUSSION TITLE
What ever happen with the k8s dashboard plugin?

### DISCUSSION THREAD

**Discussion by shaul75** [cockpit-project/cockpit#18081] [General]
This post is from way back at 2015, I can't seem to find any relevant information about it. Was that project scraped? 

https://cockpit-project.org/blog/cockpit-kubernetes-dashboard.html

Thanks

**Comment by marusak** [comment]
Since [rhel-7.5 kubernetes was replaced by openshift](https://www.redhat.com/en/blog/container-related-changes-red-hat-enterprise-linux-75). We do not implement openshift management tools as there is separate [console](https://github.com/openshift/console).

**Reply by shaul75** [reply]
Thanks @marusak.
I understand it from RH's perspective, but as Cockpit's idea is to provide a UI to interact with the underlying OS in an unopinionated way, a plugin that would just simply wrap a UI around the `kubectl` CLI would be a nice extension imo. Since it was already developed anyway, I think it's a waste not to have it live on as an open source plugin for whoever wishes to maintain/use it.

**Reply by marusak** [reply]
absolutely, it still lives in git history and it is easy to find and use as a separate plugin. Anyone is welcomed to do so and the team will happily give guidance.
However maintenance is not cheap especially if none in the team understands kubernetes well nor we have a proper testing environment.

**Comment by anapeksha** [comment]
Any update on this?

**Reply by Venefilyn** [reply]
What do you want an update on? To quote Matej from 2023

> Since [rhel-7.5 kubernetes was replaced by openshift](https://www.redhat.com/en/blog/container-related-changes-red-hat-enterprise-linux-75). We do not implement openshift management tools as there is separate [console](https://github.com/openshift/console).

and 

> [...] it still lives in git history and it is easy to find and use as a separate plugin. Anyone is welcomed to do so and the team will happily give guidance.
> However maintenance is not cheap especially if none in the team understands kubernetes well nor we have a proper testing environment.
